test/e2e/e2e_helpers_test.go

```diff
@@ -421,10 +422,91 @@ func sortNamespacedNames(nns []types.NamespacedName) {
 	})
 }
 
+// Note: This method is currently not being used in the E2E test.
+/*
 func testDataPath(description ginkgo.GinkgoTestDescription) string {
 	path := filepath.Join(description.ComponentTexts...)
 	return strings.ReplaceAll(path, " ", "_")
 }
+*/
+
+// Get unstructured object with Kuberentes dynamic clients.
+func GetUnstructuredObject(ctx context.Context, d dynamic.Interface, gvr schema.GroupVersionResource,
+	resource types.NamespacedName) (*unstructured.Unstructured, error) {
+	var (
+		unstructuredObject *unstructured.Unstructured
+		err                error
+	)
+
+	unstructuredObject, err = d.Resource(gvr).Namespace(resource.Namespace).Get(ctx, resource.Name, metav1.GetOptions{})
```

Comment:
As far as I know, it's more like the "GO way" to do this by:
```suggestion
	unstructuredObject, err := d.Resource(gvr).Namespace(resource.Namespace).Get(ctx, resource.Name, metav1.GetOptions{})
```

Same applies to all similar var declarations in this PR.

Reply:
Thanks for the comment. Addressed it.